nucliadb/nucliadb/common/cluster/manager.py

```diff
@@ -268,9 +271,13 @@ async def create_shard_by_kbid(
 
         # Append the created shard and make `actual` point to it.
         kb_shards.shards.append(shard)
+        # B/c with Shards.actual
```

Comment:
WDYT about adding unit tests to this part of the code? It's a pretty important part of nucliadb

Reply:
Added a unit test for shard creation